server/database.sql

```diff
@@ -13,21 +13,28 @@ CREATE TABLE users(
 CREATE TABLE blogs(
     id uuid PRIMARY KEY DEFAULT
     uuid_generate_v4(),
+    user_id VARCHAR(255) NOT NULL,
     user_name VARCHAR(255) NOT NULL,
     question VARCHAR(255) NOT NULL,
     blog_text VARCHAR(255) NOT NULL,
-    upvotes INT NOT NULL,
-    downvotes INT NOT NULL,
     blog_date  VARCHAR(255) NOT NULL
 );
+-- reactions is table for upvotes and downvotes
+CREATE TABLE reactions(
+    id uuid PRIMARY KEY DEFAULT
+    uuid_generate_v4(),
+    blog_id VARCHAR(255),
```

Comment:
Broooo not like this, USE foreign keys and ids are not varchar in this case use integer.